src/Gui/NaviCube.h

```diff
@@ -41,22 +41,18 @@ class GuiExport NaviCube {
         BottomLeftCorner,
         BottomRightCorner
     };
-    NaviCube(Gui::View3DInventorViewer* viewer) ;
+    NaviCube(Gui::View3DInventorViewer* viewer);
     virtual ~NaviCube();
     void drawNaviCube();
     void createContextMenu(const std::vector<std::string>& cmd);
     bool processSoEvent(const SoEvent* ev);
     void setCorner(Corner);
+    static QFont getDefaultSansserifFont();
     static void setNaviCubeCommands(const std::vector<std::string>& cmd);
     static void setNaviCubeLabels(const std::vector<std::string>& labels);
+
 private:
     NaviCubeImplementation* m_NaviCubeImplementation;
 };
 
-class HuuhaaClassPy : public Py::PythonExtension<HuuhaaClassPy> {
```

Comment:
what was this?? @nyholku

Reply:
I wondered how long it would take for someone to notice ;) It was an experiment that I forgot to remove from the code, it does nothing and should not be there.

Reply:
'huuhaa' is a Finnish expression that could be translate to approximately 'hogwash'.

Reply:
it took 5 years! hahaha